mathics/builtin/numbers/calculus.py

```diff
@@ -2033,7 +2033,7 @@ class NIntegrate(Builtin):
     >> NIntegrate[Exp[x],{x,-Infinity, 0},Tolerance->1*^-6, Method->"Internal"]
      = 1.
     >> NIntegrate[Exp[-x^2/2.],{x,-Infinity, Infinity},Tolerance->1*^-6, Method->"Internal"]
-     = 2.50664
+     = 2.5066...
```

Comment:
This isn't related to PyPy per se, but running mathics _withouth_ scipy. 
Here we get I think 2.50663

Reply:
The NIntegrate implementation using "Internal" as a method should not depend on scipy. Maybe ``Exp`` does?

Reply:
Maybe, I didn't investigate.